index.js

```diff
@@ -1,96 +1,103 @@
 'use strict'
 
 const { ECS } = require("@aws-sdk/client-ecs");
-const async = require('async');
 const _ = require('lodash');
 
 // Set the default region to 'us-east-1' if not already set
 const region = process.env.AWS_DEFAULT_REGION || 'us-east-1';
 
-const updater = function (options, cb) {
-  async.waterfall([
-    (next) => updater.currentTaskDefinition(options, next),
-    (currentTaskDefinition, next) => {
+/**
+ * updater
+ *
+ * Run the end-to-end image updating process on a service or task definition.
+ * @param {object} options A hash of options used when initiating this deployment
```

Comment:
```suggestion
 * @param {object} A hash of options used when initiating this deployment
```

Reply:
This is the standard Closure Compiler parameter annotation - the name is expected:

https://developers.google.com/closure/compiler/docs/js-for-compiler#tag-param

Reply:
ah it's a param sorry

Reply:
We don't actually use closure compiler at all in this project but I guess at some point Coen might have been using it to spot bugs. It's more useful on the frontend because it does a reasonable job of minification.

https://effectivetypescript.com/2023/09/27/closure-compiler/